### DISCUSSION TITLE
Zero Trust 部署流式接口问题

### DISCUSSION THREAD

**Discussion by reonokiy** [ChatGPTNextWeb/NextChat#1013] [General]
为了做用户认证，所以我用Docker部署到自己服务器上之后用Cloudflare Tunnel部署到公网，但是类似于Nginx的配置问题#386，在使用Cloudflare Zero Trust中的Tunnel之后，Cloudflare的代理服务器也会导致流式接口失效，前端不能显示打字机效果。

目前尝试过在Cloudflare中配置Cache Rules或者在Pages Rules中关闭缓存，但是都没有效果。请问有什么方法可以配置Zero Trust使得可以用流式接口？

**Comment by Yidadaa** [comment]
建议咨询 cloud flare 官方。
